subtests/docker_cli/rmi/rmi.py

```diff
@@ -239,3 +239,9 @@ def postprocess(self):
     def check_image_exists_by_id(self, image_id):
         di = DockerImages(self)
         return di.list_imgs_with_image_id(image_id)
+
+    def cleanup(self):
+        # FIXME: remove once bz1317096 is fixed
```

Comment:
Actually no, I think this is a genuine test bug by omission.  The  ``cleanup()`` method must always restore the system to it's prior state (or as near as possible/practical).  This test was failing that, the underlying bug is irrelevant IMHO, ``cleanup()`` should still do it's job.  This is partly the reason why ``garbage_check`` was added, there can be bugs in both docker and the test at the same time :grinning: